website/docs/language/expressions/references.mdx

```diff
@@ -321,9 +321,9 @@ Terraform cannot predict the value of this id until the object has been created.
 To allow expressions to still be evaluated during the plan phase, Terraform
```

Comment:
@wzyboy Thank you very much for your efforts on the PR. I would like to make a couple of additional changes to the paragraph. Would you mind updating your PR to replace this paragraph with the following:

"Terraform uses special unknown value placeholders for information that it cannot predict during the plan phase. The Terraform language automatically handles unknown values in expressions. For example, adding a known value to an unknown value automatically produces an unknown value as a result."